flink/flink.sh

```diff
@@ -22,7 +22,7 @@
 # Cloud Dataproc Image Version information:
 # https://cloud.google.com/dataproc/concepts/dataproc-versions
 
-set -euxo pipefail
+set -Eeuxo pipefail
```

Comment:
Why this flag?

Reply:
This is `errtrace` flag, removed because we don't use `trap` anyway